### ISSUE TITLE
3.8 and 4.0 beta version unable to update from DOI

### ISSUE THREAD

**Issue by mishrabk** [JabRef/jabref#2911]
JabRef version  on 

Steps to reproduce:

1. ...
2. ...
3. ...

<details>
  <summary>Log File</summary>
  
  ```
Info cannot be found
org.jabref.logic.importer.FetcherException: Connection error
	at org.jabref.logic.importer.fetcher.DoiFetcher.performSearchById(DoiFetcher.java:65)
	at org.jabref.gui.mergeentries.FetchAndMergeWorker.doInBackground(FetchAndMergeWorker.java:47)
	at org.jabref.gui.mergeentries.FetchAndMergeWorker.doInBackground(FetchAndMergeWorker.java:22)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Server returned HTTP response code: 406 for URL: https://data.crossref.org/10.2514%2F1.j050009
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source)
	at org.jabref.logic.net.URLDownload.asString(URLDownload.java:152)
	at org.jabref.logic.net.URLDownload.asString(URLDownload.java:166)
	at org.jabref.logic.importer.fetcher.DoiFetcher.performSearchById(DoiFetcher.java:55)
	... 8 more
Caused by: java.io.IOException: Server returned HTTP response code: 406 for URL: https://data.crossref.org/10.2514%2F1.j050009
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.net.HttpURLConnection.getResponseCode(Unknown Source)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(Unknown Source)
	at org.jabref.logic.net.URLDownload.openConnection(URLDownload.java:294)
	at org.jabref.logic.net.URLDownload.openConnection(URLDownload.java:302)
	... 11 more

</details>

**Comment by mishrabk** [comment]
Info cannot be found
org.jabref.logic.importer.FetcherException: Connection error
	at org.jabref.logic.importer.fetcher.DoiFetcher.performSearchById(DoiFetcher.java:65)
	at org.jabref.gui.mergeentries.FetchAndMergeWorker.doInBackground(FetchAndMergeWorker.java:47)
	at org.jabref.gui.mergeentries.FetchAndMergeWorker.doInBackground(FetchAndMergeWorker.java:22)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Server returned HTTP response code: 406 for URL: https://data.crossref.org/10.2514%2F1.j050009
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source)
	at org.jabref.logic.net.URLDownload.asString(URLDownload.java:152)
	at org.jabref.logic.net.URLDownload.asString(URLDownload.java:166)
	at org.jabref.logic.importer.fetcher.DoiFetcher.performSearchById(DoiFetcher.java:55)
	... 8 more
Caused by: java.io.IOException: Server returned HTTP response code: 406 for URL: https://data.crossref.org/10.2514%2F1.j050009
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.net.HttpURLConnection.getResponseCode(Unknown Source)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(Unknown Source)
	at org.jabref.logic.net.URLDownload.openConnection(URLDownload.java:294)
	at org.jabref.logic.net.URLDownload.openConnection(URLDownload.java:302)
	... 11 more

**Comment by Siedlerchr** [comment]
See #2898 Fixed with #2889

**Comment by mishrabk** [comment]
Working fine with new snapshot(JabRef_windows-x64_4_0_0-dev--snapshot--2017-06-15--master--0c006c1e1).
